### DISCUSSION TITLE
OP老大，我是X96 MAX ,能麻烦您给我的dtb里加上无线吗？

### DISCUSSION THREAD

**Discussion by The4thWall** [ophub/amlogic-s9xxx-armbian#2240] [Amlogic] [answered]
我使用的这个系统，https://github.com/kuoruan/Build-Armbian/releases/download/v5.99-20200408/Armbian_5.99_Aml-g12_Debian_buster_default_5.3.0_minimal.img.xz，比较简洁，速度也比较快，但是无法使用无线，我是新版X96 MAX，如下：
Android 9: img x96Max_311R_P
主板型号： Q5X2_LP3 19215
WIFI芯片： Fn-Link 6222B-SRB（含蓝牙V4.1）
6222B-SRB采用瑞昱RTL8822BS-CG主芯片？
无线驱动使用 https://packages.debian.org/buster/all/firmware-realtek/download，装上后rtlwifi和rtw88内有8822对应文件，但是nmtui看不到wlan，nmcli radio,WIFI-HW，WIFI，WWAN-HW，WWAN，均显示enabled

恳请帮助解决，感谢！

**Comment by ophub** [comment, marked as answer]
内核源码里没有驱动

**Reply by The4thWall** [reply]
是的，没有驱动，我到(https://packages.debian.org/buster/all/firmware-realtek/download)
，这里下载安装了驱动。 
[meson-g12a-x96-max.zip](https://github.com/ophub/amlogic-s9xxx-armbian/files/14979986/meson-g12a-x96-max.zip)
这里是系统里的dtb文件。

**Reply by The4thWall** [reply]
安装驱动后仍然在armbian里看不到无线，麻烦指教一二

**Reply by JustinTangChina** [reply]
按照大佬们的意思 这个8822的wifi官方是没有提供驱动的，你看到的驱动都是基于第三方的，所以，用ophub的系统 能驱动上能驱动起来你就用，不能用你就放弃 除非你找大神给你重写驱动

**Reply by The4thWall** [reply]
我这个不是驱动问题吧，我有驱动，是dtb里没无线的信息吧？op大佬的系统，重启或者更新了无线网卡会消失，不知道怎么回事
